python/hail/utils/misc.py

```diff
@@ -20,3 +20,11 @@ def get_env_or_default(maybe, envvar, default):
     import os
 
     return maybe or os.environ.get(envvar) or default
+
+def get_export_type(et):
```

Comment:
can we move this into Scala?

Reply:
Since we're not supporting a Scala interface, it makes more sense to me to push this kind of thing there, especially since doing anything between the two is super slow.

This function calls into the jvm 9 times, I think. That could take 30ms, and just to process one argument!